semconv/v1.3.0/resource.go

```diff
@@ -0,0 +1,774 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated from semantic convention specification. DO NOT EDIT.
+
+package semconv // import "go.opentelemetry.io/otel/semconv"
```

Comment:
Do we need to change the package name in the template or can they all be named `semconv` and distinguished by import path?  The canonical import path comment needs to be removed or updated in any event.  I was thinking about how this can be accomplished, I'm not sure if the template runner allows specifying additional values on the CLI, but if it does then we could put a `{{package_name}}` replacement in here.  If it doesn't, we could use `[[PACKAGE]]` and handle it with the rest of the replacements in `semconv-gen`.

Reply:
Package name probably fine as is since there will be an alias when importing anyway: `import semconv "go.opentelemetry.io/otel/semconv/v1.3.0"`. They can be all named semconv since they are in different directories.
Even if we wanted to we cannot name give it a name that exactly matches the version number, `v1.3.0` is not a valid package name.
So the only think we probably need to fix here is the comment so that it shows the right import path.

Reply:
I fixed the import path in the comment to be correct.